### PR TITLE
Replace unicode quotes by ascii versions

### PR DESCRIPTION
**Pull request by manolo**
[Review on Reviewable](https://reviewable.io/reviews/gwtproject/gwt-site/100)

### CODE REVIEW

**Comment by jDramaix** [comment]
What is the problem with the unicode quotes ?

File: src/main/markdown/doc/latest/polymer-tutorial/create.md [hunk 1]
```diff
@@ -45,13 +45,13 @@ For the **TodoList** project, we’ll need to run `webAppCreator` with the follo
 
 3. Launch your Browser.
 
-    In the GWT developer window, press “Launch Default Browser” to launch the application. Alternatively, you can click “Copy to Clipboard” and paste the URL into any browser.
+    In the GWT developer window, press """Launch Default Browser""" to launch the application. Alternatively, you can click """Copy to Clipboard""" and paste the URL into any browser.
```
Comment (tbroyer) [inline src/main/markdown/doc/latest/polymer-tutorial/create.md]: What do triple-doublequotes mean? (do they have special processing?)

Comment (manolo) [inline src/main/markdown/doc/latest/polymer-tutorial/create.md]: does mean nothing special, it seems like I did some extra replacement with sed

**Comment by manolo** [comment]
@jDramaix There is no a big problem since we have the appropriate meta. But editing the file in command line with tools like `vi` or using search/replace makes unicode character more difficult to use.

**Comment by manolo** [comment]
@jDramaix Actually there is a problem, those unicode characters are displayed erroneously in production, so this fix is needed:

screen shot 2015-07-18 at 09 18 35

**Comment by manolo** [comment]
@jDramaix Server is sending erroneous headers (iso-8859-1 instead of utf-8:

screen shot 2015-07-18 at 09 24 07
)

**Comment by jDramaix** [comment]
so we have to fix the server in order to send the correct charset header

**Comment by manolo** [comment]
Already sent a fix in gwt-site-webapp: https://gwt-review.googlesource.com/#/c/13193/

But, I'd rather that these pages use normal quotes because in English, ascii is the normal way to write double-quote and apostrophe.

I insist on the fact that these unicode characters are here because of a mistake copying and pasting from gdocs, instead of writing the directly with 'vi' or any other text editor.

**Comment by tbroyer** [comment]
+1 to merge this PR

**Comment by jDramaix** [comment]
No I don't have any problems :) I wanted just that the problem with the header is fixed too and it's done so everything is ok for me.

**Pull request merged**